apps/src/lib/kits/maker/boards/microBit/MicroBitBoard.js

```diff
@@ -96,6 +98,30 @@ export default class MicroBitBoard extends EventEmitter {
     return Promise.resolve()
       .then(() => this.openSerialPort())
       .then(serialPort => this.boardClient_.connectBoard(serialPort))
+      .then(() => {
+        // Delay for 0.25 seconds to ensure we have time to determine the firmware version.
```

Comment:
Nit: I wonder if we should switch out "detemine" to something like "receive" to indicate that there is not action on our end, we're just waiting to receive that info from the board?